shell/browser/api/electron_api_base_window.cc

```diff
@@ -1161,6 +1161,7 @@ void BaseWindow::RemoveFromParentChildWindows() {
   if (parent_window_.IsEmpty())
     return;
 
+  v8::HandleScope handle_scope(isolate());
```

Comment:
nit: should this use `JavascriptEnvironment::GetIsolate();` for consistency rather than isolate member from the wrappable ?